### DISCUSSION TITLE
Context menu hotkey

### DISCUSSION THREAD

**Discussion by mikekazakov** [mikekazakov/nimble-commander#172] [Ideas]
_(migrating existing issues/tasks from the previously used internal issue tracker)_ 
It'd be great to have a shortcut to open the MacOS Context menu (right mouse button) or at least the „Open with“ menu.

**Comment by okyr-ux** [comment]
Especially the „Open with“ menu!

That's the only feature I miss in Nimble Commander compared to [Marta](https://marta.sh/).

Thank you!

**Reply by mikekazakov** [reply]
It was done in v1.7, no?  
https://github.com/mikekazakov/nimble-commander/blob/main/WHATS_NEW.md#version-170-18-dec-2024

**Reply by okyr-ux** [reply]
Yes, absolutely! Sorry. I am actually using it for a couple days now.

I meant the hotkey for the **Open with** menu, but having the hothey for the context menu is good enough.
